example.php

```diff
@@ -1,7 +1,14 @@
-<?php
-
-$foo = NULL;
-// is_null is a function that does stuff 
-is_null($foo);
-
-?>
\ No newline at end of file
+// is_null your comment
<// is_null your comment
?// is_null your comment
p// is_null your comment
h// is_null your comment
p// is_null your comment
+// is_null your comment

```

Comment:
```diff
- // is_null your comment
+ // is_null {{ .funcComment }}
```

Change `{{ .funcComment }}` by commenting: `@CodeLingoBot rewrite set funcComment "your comment"`

Reply:
@CodelingoBotTester rewrite set funcComment "your comment"

Reply:
@CodelingoBotTester blah

Reply:
unexpected command 

	---
	<details>
	<summary>@CodelingoBotTester  commands and options</summary>

	You can trigger CodeLingo actions by commenting on this PR:

	+ `@@CodelingoBotTester  review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@@CodelingoBotTester  review`</summary>

	    + `@@CodelingoBotTester  review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
	    + `@@CodelingoBotTester  review un-ignore` un-ignores the issue for current and future reviews.
	+ `@@CodelingoBotTester  rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@@CodelingoBotTester  rewrite`</summary>

	    + `@CodelingoBotTester  rewrite set <varname>` sets the value of a variable in a rewrite template.


	Mention @@CodelingoBotTester  to reach us.

	</details>

Reply:
@CodelingoBotTester blah

Reply:
unexpected command 

		---
		<details>
		<summary>CodeLingoBot commands and options</summary>

		You can trigger CodeLingo actions by commenting on this PR:`@CodelingoBotTester  review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`%!s(MISSING) review`</summary>`%!s(MISSING) review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.`%!s(MISSING) review un-ignore` un-ignores the issue for current and future reviews.`%!s(MISSING) rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@%!s(MISSING) rewrite`</summary>`%!s(MISSING) rewrite set <varname>` sets the value of a variable in a rewrite template.


		Mention %!s(MISSING) to reach us.

		</details>

Reply:
@CodelingoBotTester blah

Reply:
unexpected command 
	---
	<details>
	<summary>CodeLingoBot commands and options</summary>
	You can trigger CodeLingo actions by commenting on this PR:
	+ `@CodeLingoBot review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodeLingoBot review`</summary>
	    + `@CodeLingoBot review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
	    + `@CodeLingoBot review un-ignore` un-ignores the issue for current and future reviews.
	+ `@CodeLingoBot rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodeLingoBot rewrite`</summary>
	    + `CodeLingoBot rewrite set <varname>` sets the value of a variable in a rewrite template.
	Mention @CodeLingoBot to reach us.
	</details>

Reply:
@CodelingoBotTester blah

Reply:
unexpected command 

---
<details>
<summary>CodeLingoBot commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodeLingoBot review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodeLingoBot review`</summary>
    + `@CodeLingoBot review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodeLingoBot review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodeLingoBot rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodeLingoBot rewrite`</summary>
    + `CodeLingoBot rewrite set <varname>` sets the value of a variable in a rewrite template.
Mention @CodeLingoBot to reach us.
</details>

Reply:
unexpected command 

---
<details>
<summary>CodeLingoBot commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodeLingoBot review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodeLingoBot review`</summary>
    + `@CodeLingoBot review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodeLingoBot review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodeLingoBot rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodeLingoBot rewrite`</summary>
    + `CodeLingoBot rewrite set <varname>` sets the value of a variable in a rewrite template.
Mention @CodeLingoBot to reach us.
</details>

Reply:
unexpected command 

---
<details>
<summary>CodeLingoBot commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodeLingoBot review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodeLingoBot review`</summary>
    + `@CodeLingoBot review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodeLingoBot review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodeLingoBot rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodeLingoBot rewrite`</summary>
    + `CodeLingoBot rewrite set <varname>` sets the value of a variable in a rewrite template.
Mention @CodeLingoBot to reach us.
</details>

Reply:
@CodelingoBotTester blah

Reply:
unexpected command 

---
<details>
<summary>CodeLingoBot commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodelingoBotTester  review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  review`</summary>
    + `@CodelingoBotTester  review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodelingoBotTester  review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodelingoBotTester  rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  rewrite`</summary>
    + `@CodelingoBotTester  rewrite set <varname>` sets the value of a variable in a rewrite template.
Mention @CodelingoBotTester  to reach us.
</details>

Reply:
unexpected command 

---
<details>
<summary>CodeLingoBot commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodelingoBotTester  review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  review`</summary>
    + `@CodelingoBotTester  review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodelingoBotTester  review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodelingoBotTester  rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  rewrite`</summary>
    + `@CodelingoBotTester  rewrite set <varname>` sets the value of a variable in a rewrite template.
Mention @CodelingoBotTester  to reach us.
</details>

Reply:
@CodelingoBotTester blah

Reply:
unexpected command 

---
<details>
<summary>CodeLingoBot commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodelingoBotTester  review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  review`</summary>
    + `@CodelingoBotTester  review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodelingoBotTester  review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodelingoBotTester  rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  rewrite`</summary>

    + `@CodelingoBotTester  rewrite set <varname>` sets the value of a variable in a rewrite template.

Mention @CodelingoBotTester  to reach us.

</details>

Reply:
unexpected command 

---
<details>
<summary>CodeLingoBot commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodelingoBotTester  review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  review`</summary>
    + `@CodelingoBotTester  review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodelingoBotTester  review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodelingoBotTester  rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  rewrite`</summary>

    + `@CodelingoBotTester  rewrite set <varname>` sets the value of a variable in a rewrite template.

Mention @CodelingoBotTester  to reach us.

</details>